docs/installation.md

```diff
@@ -12,6 +12,61 @@
 
 Follow detailed instructions in separate [Domain Protect Deploy](https://github.com/domain-protect/domain-protect-deploy) repository with GitHub Actions deployment workflow
 
+## Atlantis
+
```

Comment:
Suggest a one line intro for those (like myself) not familiar with Atlantis, like you put in on the top level README, e.g:
[Atlantis](https://www.runatlantis.io) is a tool for Terraform pull request automation

Reply:
updated